healthcare/ich_segmentation/cassandra/cql/cqlshrc

```diff
@@ -0,0 +1,11 @@
+[authentication]
+username = user
+password = password
```

Comment:
the same here, please remove hardcoded option, we dont want our users to use these files as is, with hardcoded authentication info.

Reply:
Same case as jmx examples.